spec/models/service_provider_spec.rb

```diff
@@ -124,7 +124,52 @@
       end
     end
   end
+  describe '#semantic_authn_contexts_allowed?' do
+    context 'when the semantic authn contexts feature is enabled' do
+      before do
+        allow(IdentityConfig.store).
+          to receive(:feature_valid_authn_contexts_semantic_enabled).
+          and_return(true)
+      end
+      context 'when the service provider is in the allowed list' do
```

Comment:
[nitpick] can you add newlines before context blocks for readability please? thanks!

Reply:
weird, i expected rubocop would've fixed those, but i guess not.

Reply:
i don't think it's a hard rule we currently have, i just find it much easier to read/scan long spec files with those kinds of newline breaks. to be clear, this is a nitpicky davida preference :)

Reply:
Fixed